sections/main-collection-banner.liquid

```diff
@@ -14,7 +24,7 @@
     </div>
 
     {%- if section.settings.show_collection_image and collection.image -%}
-      <div class="collection-hero__image-container media">
+      <div class="collection-hero__image-container media gradient">
```

Comment:
Good catches @melissaperreault 🙂  thank you. 
One of the issue isn't related to this fix, so we can create a different issue to tackle it. 

As for the border image, I added the `gradient` class it was missing and now the border is coming up with the right color matching what we have for the image with text section. 👍 

Which made me test a scenario with a gradient for the background and realized the collage section is missing that class too:
![](https://screenshot.click/22-01-5qzxf-kydjz.png)

But that can be created as an issue or actually, I will add this tidbit in the existing issue for Collage.

Reply:
Thanks Ludo!